added ender pearls to crushed end stone to get as a resource

raziel23x opened the pull request.

raziel23x committed:
Create ender_pearl.json

NovaMachina reviewed (changes requested):
Please also add the recipe to the data generators. Found under Recipes#registerSieveRecipes

raziel23x committed:
moved to datagen as requested

raziel23x committed:
fixed derp

raziel23x committed:
forgot runData

raziel23x commented:
since i made i moved it from the data folder to the datagen#Recipes#registerSieveRecipes i deleted the original since you only need to datagen version

raziel23x commented:
> Please also add the recipe to the data generators. Found under Recipes#registerSieveRecipes

done
